packages/serverpod/lib/src/server/websocket_request_handlers/method_websocket_request_handler.dart

```diff
@@ -121,25 +121,8 @@ class MethodWebsocketRequestHandler {
     WebSocket webSocket,
     OpenMethodStreamCommand message,
   ) async {
-    // Validate targeted endpoint method
-    var endpointConnector =
-        server.endpoints.getConnectorByName(message.endpoint);
-    if (endpointConnector == null) {
-      server.serverpod.logVerbose(
-        'Endpoint not found for open stream request: $message',
-      );
-      return OpenMethodStreamResponse.buildMessage(
-        connectionId: message.connectionId,
-        endpoint: message.endpoint,
-        method: message.method,
-        responseType: OpenMethodStreamResponseType.endpointNotFound,
-      );
-    }
-
-    // Create session
     var session = MethodStreamSession(
       server: server,
-      enableLogging: endpointConnector.endpoint.logSessions,
```

Comment:
Comment: Can we inject this variable into the method instead? I still believe centralizing this logic inside the session is better than spreading the logging init logic around the code base.

Reply:
Refactored into a callback instead: [cec5fad](https://github.com/serverpod/serverpod/pull/2676/commits/cec5fad3d1dececc42a205ca6662da6c4342f464)